Annotato/Annotato/Persistence/PersistenceService/OfflinePersistenceService/OfflinePersistenceService+AnnotationsPersistence.swift

```diff
@@ -16,4 +16,9 @@ extension OfflinePersistenceService: AnnotationsPersistence {
         annotation.setDeletedAt(to: Date())
         return await localPersistence.annotations.deleteAnnotation(annotation: annotation)
     }
+
+    func createOrUpdateAnnotationsForLocal(annotations: [Annotation]) -> [Annotation]? {
+        fatalError("OfflinePersistenceService::createOrUpdateAnnotationForLocal: This function should not be called")
+        return nil
```

Comment:
This can be implemented. It's just `localPersistence.createOrUpdateAnnotations(annotations: annotations)`